cfme/tests/test_ui_problems.py

```diff
@@ -12,7 +12,8 @@
 TOLERANCE = 20
 # TODO: Replace nav branch strings with navmazing object destinations when supported
 LOCATIONS = [
-    "control_explorer", "automate_explorer", "automate_customization", "my_services",
+    "control_explorer", "automate_explorer", "automate_customization",
```

Comment:
I already do this is one place already so maybe we should make a consolidated function in a common area that takes either a string or a tuple and blah blah